src/importlinter/contracts/layers.py

```diff
@@ -24,6 +24,7 @@ class ModuleTail:
 @dataclass(frozen=True)
 class Layer:
     module_tails: set[ModuleTail]
+    independent: bool
```

Comment:
Could we rename to `is_independent` and default this to `True`?

Reply:
done (although in grimp we called it just `independent`)

Reply:
Hmm, that's a good point. I'm in the habit of naming booleans like this, but perhaps it's better to have it the same as Grimp. I'll leave it up to you whether you want to change it back to just `independent`, I don't mind too much.